docs/userGuide/contentAuthoringExamples.md

```diff
@@ -0,0 +1,56 @@
+<link rel="stylesheet" href="{{baseUrl}}/css/main.css">
+<style>
+    .radio-list-item,
+    .task-list-item {
+        margin-left: 0;
+    }
+</style>
+
+<include src="../common/header.md" />
+
+<div class="website-content">
+
+# Examples
+
+## Question body syntax
+
+MarkBind allows for the easy use of checkboxes and radio buttons with the Question and Panel components in [VueStrap modified by MarkBind](https://markbind.github.io/vue-strap/).
```

Comment:
"MarkBind allows ~~for the easy use of~~ **using** checkboxes and radio buttons with..."